notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -814,7 +804,8 @@
         "    job_display_name=f\"iowa_liquor_sales_forecasting_predictions_{TIMESTAMP}\",\n",
         "    bigquery_source=PREDICTION_DATASET_BQ_PATH,\n",
         "    instances_format=\"bigquery\",\n",
-        "    bigquery_destination_prefix=batch_predict_bq_output_uri_prefix,\n",
+        "    # bigquery_destination_prefix=batch_predict_bq_output_uri_prefix,\n",
```

Comment:
Seems that this line needs to be reverted to maintain original purpose of showing a bq destination.

Reply:
The batch_predict can do either or, not both - as the reset of the rest of the notebook uses GCS, I opted with it (I added some markup to explain the difference)